components/telemetry-operator/internal/helper/defaults.go

```diff
@@ -0,0 +1,62 @@
+package helper
```

Comment:
I'd prefer to have a segment in the package name that indicates that this `helper` is not a generic helper, but a `logpipeline` helper. For example, here some good example: https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/deployment/util/deployment_util.go.

Reply:
Agree as discussed, but let's adapt the structure of internal pkg in a separate PR. 👌